Changelog.md

```diff
@@ -1,8 +1,10 @@
-## 1.3.2 - UNRELEASED
+## 1.4.0 - 2024-04-16
 
 ### Changed
 
 -   Updated documentation links to point to the TechDocs platform.
+-   Moved feedback tab to a dialog which can be opened by going to the about tab
+    and click Give Feedback.
```

Comment:
```suggestion
-   Reworked the Feedback tab to be a dialog in the About tab (Give Feedback).
```

Reply:
I feel I talked about this to @kylebonnici , we agreed on the change of wording, but then the edits never made it to the repo -.-. Anyway, this is what I suggest now.